networking/spoke-BU0001A0008.bicep

```diff
@@ -266,6 +329,17 @@ resource vnetSpoke 'Microsoft.Network/virtualNetworks@2021-05-01' = {
           privateLinkServiceNetworkPolicies: 'Disabled'
         }
       }
+      {
+        name: 'snet-privatelinkendpoints'
+        properties: {
+          addressPrefix: '10.240.4.32/28'
+          networkSecurityGroup: {
+            id: nsgPrivateLinkEndpointsSubnet.id
+          }
+          privateEndpointNetworkPolicies: 'Disabled'
+          privateLinkServiceNetworkPolicies: 'Enabled'
```

Comment:
I can't leave a comment on this line, but above in `snet-clusternodes`, this leaves `privateLinkServiceNetworkPolicies: 'Enabled'` -- we'll probably want to set that back to `privateLinkServiceNetworkPolicies: 'Disabled'` now that we don't have any private link endpoints showing up there.

Reply:
Updated the comment